.circleci/config.yml

```diff
@@ -192,12 +192,20 @@ workflows:
 
       - deploy:
           name: staging_deploy_live_1
-          namespace: staging
+          namespace: staging-live1
```

Comment:
Same query as [on the FALA PR](https://github.com/ministryofjustice/fala/pull/108#discussion_r760342483)

Reply:
Thanks, that's a really good suggestion.

Updated now to reflect this feedback